stable/kubernetes-dashboard/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: kubernetes-dashboard
-version: 1.5.2
+version: 1.5.3
```

should be
```suggestion
version: 1.6.0
```
not a bug fix